hash_table.h

```diff
@@ -392,6 +392,20 @@ static void PREFIX(_remove)(PREFIX(_table) *table, void *key)
 	MAP_LOCK();
 	PREFIX(_table_cell) cell = PREFIX(_table_get_cell)(table, key);
 	if (NULL == cell) { return; }
+
+	uint32_t hash = MAP_TABLE_HASH_KEY(key);
```

Comment:
@ms-jihua roughly: negligible. The hash function in use here takes a 32-bit value and shifts it around to return another 32-bit value, and `table_get_cell` is practically just `return &table[cell%size]`.